specification/components/stu3/examples/PasNotRespondingOperationOutcome.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
As for r4 the examples are copied from the sandbox during the build, so this directory should b added to .gitignore